docs/faq.md

```diff
@@ -20,6 +20,17 @@ You should look for places in your code that:
   - use an algorithm that has two or more implementations (to verify their equivalence).
   - look for existing fuzz target [examples](https://github.com/google/oss-fuzz/tree/master/projects) and find similarities.
 
+## My project is not open source. Can I use OSS-Fuzz?
+
+You cannot use OSS-Fuzz, but you can use [ClusterFuzz] which OSS-Fuzz is based
```

Comment:
s/ClusterFuzz is a fuzzing intrastructure/ClusterFuzz is a fully open-source fuzzing infrastructure that you can deploy in your own environment and run continuously at scale.

Move "OSS-Fuzz is a production instance" to a new para and remove last line "You can deploy your own instance of
ClusterFuzz and use it for fuzzing at scale."

Reply:
Done, but I've removed word "fully", as CF relies on GCP and someone might complain that not every single bit is open source.